compass/sign/RemoteSignatureSource.java

```diff
@@ -56,11 +57,36 @@ public RemoteSignatureSource(String uri,
    * @param uri the URI of the host to connect to
    */
   public RemoteSignatureSource(String uri) {
-    this(ManagedChannelBuilder.forTarget(uri).usePlaintext().idleTimeout(5, TimeUnit.SECONDS).build());
+    this.channelBuilder = createPlaintextManagedChannelBuilder(uri);
+    this.serviceStub = SignatureSourceGrpc.newBlockingStub(channelBuilder.build());
+  }
+
+  private ManagedChannelBuilder createSecureManagedChannelBuilder(String uri,
+                                                                  String trustCertCollectionFilePath,
+                                                                  String clientCertChainFilePath,
+                                                                  String clientPrivateKeyFilePath) throws SSLException {
+    String cacheTtl = Security.getProperty("networkaddress.cache.ttl");
+    if (cacheTtl == null) {
+      cacheTtl = "5";
```

Comment:
Sorry for being a hard-ass

If we want to change the value of this number, we need to do it in `main`  and 15 lines below.
Another dev that is not Andrea (let's call him Gal) may forget to change in all places.

Can you please create a `public static final` in this class?
Thanks

Reply:
Make complete sense, sorry for this.

Reply:
Makes total sense! Sorry for that